src/pages/storey/containers.mdx

```diff
@@ -43,6 +43,32 @@ container.
 
 To avoid key collisions, you must provide a different byte to each container.
 
+# Accessor
+
+Every container has a corresponding "accessor" type that is used to perform all read/write
+operations.
+
+To get the accessor of a top-level container, the [`access` method] must be called. It takes a reference
+to a storage backend.
+
+```rust template="storage"
+use cw_storey::containers::Item;
+use cw_storey::CwStorage;
+
+let item: Item<u32> = Item::new(0);
+
+item.access(&mut CwStorage(&mut storage))
+    .set(&42)
+    .unwrap();
+
+assert_eq!(
+    item.access(&CwStorage(&storage))
+        .get()
+        .unwrap(),
+    Some(42)
```

Comment:
Hmm, this is two levels of unwrapping. Is this really needed?

Reply:
It felt more like idiomatic Rust to me. `Option` is usually how you represent a possibly-empty value, `Result` is more for fallibility due to read errors or w/e. Also, this mimicks the `may_load` API from `storage-plus`.

Reply:
@webmaster128 also, it is fairly easy to convert it to a single option with `result.ok().flatten()`. If you want to make it into a single result its more work.
Maybe we could add an additional function named `try_get` that will return straight `Result<T, Err>` with an error when the item is missing - for the cases when we don't wan to separately handle the missing item.

Reply:
I think I like that. The more explicit thing being a default feels better.